src/commands/install_sdk.yml

```diff
@@ -1,6 +1,10 @@
 description: >
   Install Flutter SDK
 parameters:
+  channel:
+    type: string
```

Comment:
More a question than a request, are these channels static and stable?  If so, it could be good to have the type as enum with the possible channels. 
If they add, update or remove channels very often, then string is good.

Reply:
I think they will prob only be `stable` and `beta`. It has been that way for a very long time.